tests/scripts/thread-cert/pktverify/packet_filter.py

```diff
@@ -454,13 +454,19 @@ def filter_LLANMA(self, **kwargs):
     def filter_LLABMA(self, **kwargs):
         return self.filter(lambda p: p.ipv6.dst == consts.LINK_LOCAL_ALL_BBRS_MULTICAST_ADDRESS, **kwargs)
 
+    def filter_LLARMA(self, **kwargs):
```

Comment:
nit: filter_LLARMA_dst

Reply:
Maybe not necessary to use `_dst` because a message should never use a multicast address can only be the destination but never the source?

Reply:
yes, my intention is that adding _dst could be more readable